c++/src/capnp/layout.c++

```diff
@@ -841,22 +845,24 @@ struct WireHelpers {
           case ElementSize::INLINE_COMPOSITE: {
             auto wordCount = ref->listRef.inlineCompositeWordCount();
             KJ_REQUIRE(boundsCheck(segment, ptr, wordCount + POINTER_SIZE_IN_WORDS),
-                       "Message contained out-of-bounds list pointer.") {
+                       "Message contained out-of-bounds list pointer."
+                       OUT_OF_BOUNDS_ERROR_DETAIL) {
               return result;
             }
 
             const WirePointer* elementTag = reinterpret_cast<const WirePointer*>(ptr);
             auto count = elementTag->inlineCompositeListElementCount();
 
             KJ_REQUIRE(elementTag->kind() == WirePointer::STRUCT,
-                       "Don't know how to handle non-STRUCT inline composite.") {
+                       "Schema mismatch: Don't know how to handle non-STRUCT inline composite.") {
```

Comment:
Same here, not a schema mismatch, the underlying data is not valid under any schema.